### DISCUSSION TITLE
第6节课笔记(2班)

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/Tutorial#410] [笔记]
## 2班第六节课笔记交这里哟~
* 链接形式提交
* 也记得在信息收集文档对应作业处填写链接

**Comment by ZixinxinWang** [comment]
https://blog.csdn.net/Wzxdecsdn/article/details/135711049

**Comment by RyanWu31** [comment]
https://github.com/RyanWu31/tutorial_notebook/blob/main/README.md

**Comment by DarthThomas** [comment]
[第六课笔记](https://share-tianhe.notion.site/d2b4a435ae6e4cf88f26571007fb223d?pvs=4)

**Comment by songyuc** [comment]
[《OpenCompass 大模型评测》学习笔记](http://t.csdnimg.cn/cRTLG)

**Comment by CoderJackZhu** [comment]
[OpenCompass 大模型评测笔记](https://zhuanlan.zhihu.com/p/679020245?)

**Comment by xulinui** [comment]
https://h90ag9106t.feishu.cn/docx/FpmkdNBtroX7LMxnioQcMAqJnWf

**Comment by YanxingLiu** [comment]
第六节课 https://s5eixc6gba.feishu.cn/docx/UlYzdN9QjooOokxbowVcwG1QnWf?from=from_copylink

**Comment by zyy0831** [comment]
第六节课 https://www.yuque.com/yuanyang-xb8hd/eeag2y/qxqi2trcpg5n2yhh

**Comment by pranerd** [comment]
[课程笔记](https://blog.csdn.net/gaxzj/article/details/135830056?spm=1001.2014.3001.5502)

**Comment by Mr-Vincenyang** [comment]
[课程笔记](https://zhuanlan.zhihu.com/p/679797112)

**Comment by PrinceRay7** [comment]
第六节课笔记
https://blog.csdn.net/Prince_Ray/article/details/135834024?spm=1001.2014.3001.5501

**Comment by kishiokon** [comment]
[OpenCompass 大模型评测](https://www.bilibili.com/read/cv30373720)
